app/models/create_cap_proposal.rb

```diff
@@ -0,0 +1,90 @@
+class CreateCapProposal
+  def initialize(auction)
+    if !auction.is_a?(Presenter::Auction)
+      fail "an instance of Auction::Presenter must be passed to CreateCapProposal::initialize"
+    end
+
+    @auction = auction
+  end
+
+  def configure_c2_client!
```

Comment:
I think I agree w @harrisj 's earlier comments: can we do this in `config/initializers` ?